www/source/docs/run-packages-service-groups.html.md

```diff
@@ -3,44 +3,58 @@ title: Run packages in a service group
 
-In addition, multiple service groups can reside in the same ring. This allows data exposed by supervisors to be shared with other members of the ring regardless of which group they are in.
+In addition, multiple service groups can reside in the same ring. This allows data exposed by supervisors to 
+be shared with other members of the ring regardless of which group they are in.
 
 ## Joining a service group
-To join a service group, run your service either natively with the `hab start` command, or through an external runtime format such as a Docker container. A best practice is to name the group _servicename_._environment_ to support continuous deployment workflows.
+To join a service group, run your service either natively with the `hab start` command, or 
+through an external runtime format such as a Docker container. A best practice is to name the 
+group after an environment to support continuous deployment workflows. 
 
-Here's how to start up the service using the supervisor (`hab-sup`) directly:
+> Note: When applying configuration updates, you must reference the fully-qualified service group name, _servicename_._groupname_. 
 
-    hab start myorigin/myapp --group myapp.prod
+Here's how to start up the `myapp` service with the `prod` group name using the supervisor (`hab-sup`) directly:
+
+    hab start myorigin/myapp --group prod
 
 Here's how to run the same command from a Docker container:
 
-    docker run -it myorigin/myapp --group myapp.prod
+    docker run -it myorigin/myapp --group prod
 
-You will see a similar output below when your service starts. The census entry shows which service group your service belongs to using the format _servicename_._groupname_. Keep this service instance running.
+You will see a similar output below when your service starts. The census entry shows which service 
+group your service belongs to, which in this case is **myapp.prod**. Keep this service instance running.
 
     hab-sup(MN): Starting myorigin/myapp
     hab-sup(GS): Supervisor 172.17.0.2: 38a2ac4f-5348-48df-86f0-b293284740ce
-    hab-sup(GS): Census myapp.myapp.prod: c60bfa12-c913-4f89-b4f3-d082d1310c3d
+    hab-sup(GS): Census myapp.prod: c60bfa12-c913-4f89-b4f3-d082d1310c3d
     ...
 
-Services only join together to form a ring when a peer IP address is specified by other similar services at runtime.  In a new window or external runtime format, run the same command you ran above, but this time, reference the peer IP address of the previous service specified in the supervisor output above.
+Services only join together to form a ring when a peer IP address is specified by other similar services 
+at runtime.  In a new window or external runtime format, run the same command you ran above, but this time, 
+reference the peer IP address of the previous service specified in the supervisor output above.
 
-    hab start myorigin/myapp --group myapp.prod --peer 172.17.0.2
+    hab start myorigin/myapp --group prod --peer 172.17.0.2
```

Comment:
should it be noted that the default port of `9634` is used if you don't specify one with `--peer`?